### PR TITLE
Refactor clientInfo networkService tests

### PR DESCRIPTION
**Pull request by lvfxx**

### CODE REVIEW

File: pkg/networkservice/common/clientinfo/client_test.go [hunk 1]
```diff
@@ -26,110 +26,115 @@ import (
...
+		"PodNameKey":     "BBB",
+		"ClusterNameKey": "CCC",
+		"SomeOtherLabel": "DDD",
+	}
+
+	err := setEnvs(envs)
+	assert.Nil(t, err)
+
+	client := clientinfo.NewClient()
+	conn, err := client.Request(context.Background(), &networkservice.NetworkServiceRequest{
```
Comment (edwarnicke) [inline pkg/networkservice/common/clientinfo/client_test.go]: Lets us https://github.com/networkservicemesh/sdk/blob/master/pkg/networkservice/utils/checks/checkrequest/client.go#L39 here to check to make sure the labels are set before after the client in the chain, and not mistakenly 'on return'.  Current checking the return only tells us if the labels are set on return (as done here).

Comment (lvfxx) [reply pkg/networkservice/common/clientinfo/client_test.go]: Ok, fixed it

**Comment by edwarnicke** [comment]
@lvfxx Any idea why the tests are failing?

**Comment by lvfxx** [comment]
> @lvfxx Any idea why the tests are failing?

I think i need to rebase onto `master`, this flaky test is already fixed

**Pull request merged**